tests/bluetooth/controller/ctrl_encrypt/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+    tags: test_framework bluetooth encrypt ull_llcp
+tests:
+  bluetooth.controller.ctrl_encrypt.test:
+     type: unit
+     extra_args: CONFIG_BT_CTLR_PHY=y CONFIG_BT_CTLR_PHY_CODED=y CONFIG_BT_CTLR_PHY_2M=y CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
```

Comment:
Could we remove this, or  just turn it into a comment?
Similar in the other testcase.yaml files.

Reply:
Yes, again I want to have an example of this trick easily available. I'll remove it here, and add it somehow to the feature exchange unit tests